assets/js/admin/course-general-sidebar.js

```diff
@@ -171,6 +172,24 @@ const CourseGeneralSidebar = () => {
 				/>
 			) : null }
 
+			<HorizontalRule />
+
+			<h3>{ __( 'Enrollment', 'sensei-lms' ) }</h3>
+			<CheckboxControl
+				label={ __( "Don't allow self enrollment", 'sensei-lms' ) }
```

Comment:
"self enrollment" has a hyphen "self-enrollment".

Reply:
Updated in https://github.com/Automattic/sensei/pull/7231/commits/dc9500c37976f60c7cc719d99ff2b746a6577878